package io.connorwyatt.flashcards.helpers.auth.exceptions

class UserNotFoundException : AuthException()
{
}
